task_manager/api.py

```diff
@@ -82,32 +87,31 @@ def get(self, task_id):
         return tasks.model_delete_task(task_id)
 
 
-@task_ns.route('/create', methods=['GET', 'POST'])
-class CreateUser(Resource):
-    def get(self):
-        return users.model_get_create_user()
-
-    def post(self):
-        return users.model_post_create_user()
+@user_ns.route('/<int:user_id>')
+class GetUser(Resource):
+    def get(self, user_id):
+        return users.get_user(user_id)
 
 
-@user_ns.route('/')
-@user_ns.route('/<int:user_id>')
-class FetchUser(Resource):
-    def get(self, user_id=-1):
-        return users.model_fetch_user(user_id)
+@user_ns.route('/users')
+class GetUsers(Resource):
```

Comment:
I'm not a big fan of this format

users/users as a URL doesn't look very nice vs users/ and users/{userid}

Reply:
Can we do @user_ns.route('/') instead? Or will there be an issue there